### DISCUSSION TITLE
blog?

### DISCUSSION THREAD

**Discussion by simkimsia** [simkimsia/stpy#3] [General]
@gvwilson if you don't mind, i slightly prefer discussing in public by default. If u want me to bring back to private, just say the word.

Referring to your email, 

> Though if I may offer some advice, if you include a blog in the stpy site and write a couple of sentences each time you make an update, that may prove useful down the road.)

by blog, do you have any strong preferences or recommendations?

I want to hear your thoughts before I decide. My first inclination is to pick something simple like GH pages first

**Comment by gvwilson** [comment]
Hi,

Public discussion is always my preference too, and yes, a straightforward GH Pages site is easiest - that way you can make each chapter a site page as things start to come together. (I’ve built something more complicated for the existing book, but that was partly because I wanted to learn more JavaScript…)

Cheers,
G
